### PR TITLE
Install hako-parameterstore

### PR DESCRIPTION
**Pull request by mgi166**
I would like to use [moaikids/hako-parameterstore](https://github.com/moaikids/hako-parameterstore) for decryption secret value from [AWS Systems Manager ParameterStore](https://docs.aws.amazon.com/ja_jp/systems-manager/latest/userguide/systems-manager-paramstore.html) in hako image.

### CODE REVIEW

File: hako/2.3.1/Dockerfile [hunk 1]
```diff
@@ -16,5 +16,5 @@ COPY --from=builder /opt/jsonnet/libjsonnet.h /usr/local/include/libjsonnet.h
 RUN apk --update --no-cache add openssh git zip make docker libstdc++ \
       && apk add --update --no-cache --virtual .build-deps g++ \
       && JSONNET_USE_SYSTEM_LIBRARIES=1 gem install hako -v 2.3.1 -N --no-rdoc --no-ri \
+      && gem install specific_install && gem specific_install https://github.com/moaikids/hako-parameterstore.git master
```
Comment (civitaspo) [inline hako/2.3.1/Dockerfile]: Could you use commit hash ? The result changes at every timing of building.

File: hako/2.5.0/Dockerfile [hunk 2]
```diff
@@ -19,4 +19,5 @@ ENV HAKO_VERSION=2.5.0
 RUN apk --update --no-cache add openssh git zip make docker libstdc++ \
       && apk add --update --no-cache --virtual .build-deps g++ \
       && JSONNET_USE_SYSTEM_LIBRARIES=1 gem install hako -v ${HAKO_VERSION} -N --no-rdoc --no-ri \
+      && gem install specific_install && gem specific_install https://github.com/moaikids/hako-parameterstore.git master
```
Comment (civitaspo) [inline hako/2.5.0/Dockerfile]: ditto

File: hako/2.3.1/Dockerfile (hunk 1 above)
Comment (mgi166) [reply hako/2.3.1/Dockerfile]: Sure, Fix it.

File: hako/2.5.0/Dockerfile [hunk 3]
```diff
@@ -19,4 +19,5 @@ ENV HAKO_VERSION=2.5.0
 RUN apk --update --no-cache add openssh git zip make docker libstdc++ \
       && apk add --update --no-cache --virtual .build-deps g++ \
       && JSONNET_USE_SYSTEM_LIBRARIES=1 gem install hako -v ${HAKO_VERSION} -N --no-rdoc --no-ri \
+      && gem install specific_install -N && gem specific_install https://github.com/moaikids/hako-parameterstore.git -r 8ed6452
```
Comment (mgi166) [inline hako/2.5.0/Dockerfile]: `8ed6452` is https://github.com/moaikids/hako-parameterstore/commit/8ed6452bd15e24c01b0143de6bccb5bfa86942a4

**Comment by koid** [comment]
👍

**Pull request merged**